queries/pois.jinja2

```diff
@@ -7,9 +7,7 @@ SELECT
     way_area AS area,
     tags->'protect_class' AS protect_class,
     tags->'operator' AS operator,
-    tags->'cuisine' AS cuisine,
     tags->'religion' AS religion,
-    tags->'sport' AS sport,
```

Comment:
Please don't remove the general sports tag now (but it's okay to remove the general cuisine tag).

I think there are a few more kinds that have useful sports tags. Let's not take sport out until we've sorted out which is which, and that work is bucketed into https://github.com/tilezen/vector-datasource/issues/1008 in the next release.